package.json

```diff
@@ -11,6 +11,7 @@
     "type": "git",
     "url": "https://github.com/jonschlinkert/window-size.git"
   },
+  "bin": "cli.js",
```

Comment:
great! could you also add `cli.js` to the `files` array in package.json? thanks!